detectable/src/main/java/com/blackduck/integration/detectable/detectables/gradle/inspection/parse/GradleReportLineParser.java

```diff
@@ -39,7 +42,8 @@ public class GradleReportLineParser {
 
     // This map is handling all the child-parent relationships which are found in the entire project
     // with each child as a key and their respective parent as the value.
```

Comment:
This comment is not as accurate anymore. Would be good to update it and check if any other comments need updating.

Reply:
Yes sorry, should have put the PR into draft, going through that right now.